newjersey/navigator.business.nj.gov
# feat: [#187637099] update myNJ logo

## Description



We're swapping out the myNewJersey logo that appears on [/account-setup](http://localhost:3000/account-setup) to the new myNewJersey logo.

### Ticket



This pull request resolves [#187637099](https://www.pivotaltracker.com/story/show/187637099).

### Approach



Since we're not specifying the image's height and width anywhere, it's just a simple image replacement.

### Steps to Test



1. Get past onboarding.
2. Click the link in the sidebar to link your account with a myNewJersey account.
3. Observe the new logo.

### Notes



We may want to reach out to OIT to get a higher-resolution logo (or an .svg) for this.

## Code author checklist
